internal/updater/updater.go

```diff
@@ -88,7 +88,7 @@ func (u *AvailableUpdate) prepareInstall(installTargetPath string, args []string
 	}
 
 	if installTargetPath == "" {
-		installTargetPath, err = installation.InstallPath()
+		installTargetPath, err = installation.InstallPathFromArg0()
```

Comment:
How will this get the correct install path when a user is just running `state update`? I do not think `installTargetPath` is normally set at this point. It is in the bugged version but other versions I've tested it is not.

Reply:
It would get it from the `state` executable that it's calling from: http://github.com/ActiveState/cli/blob/remove-legacy-installer/internal/installation/paths.go#L46-L46